config/metricConfigs/learnMetrics.ts

```diff
@@ -166,6 +166,42 @@ const allowedMetrics: MetricsConfig = [
       },
     ],
     protocol: "statsd"
+  },
+  {
+    name: "learn_hub__time_to_interactive_dashboard_threshold",
+    help: "Count the number of page loads under the threshold",
+    type: "counter",
+    labels: [
+      {
+        name: "appName",
+        allowedValues: ["/learn"]
+      },
+      {
+        name: "pageName",
+        allowedValues: ["dashboard"]
+      },
+      {
+        name: "underThreshold",
+        allowedValues: ["true", "false"]
```

Comment:
Do we send a string or boolean from the learn-hub frontend?

Reply:
(not sure if it makes a difference)

Reply:
it seems like everything there is a string, so I assumed that the prometheus client will cast everything. All the numbers and booleans seem to be set as strings in this file.